### PR TITLE
Add 'login' page to 'system' menu

### PR DESCRIPTION
**Pull request by jcapona**
| Status  | Ticket/Issue |
| :---: | :--: |
| Ready | [Ticket](https://pi-top.atlassian.net/browse/OS-1215) |

#### Main changes
- Adds 'login' page with username and password to the 'network' menu

#### Screenshots (feature, test output, profiling, dev tools etc)

image

#### Other notes (e.g. implementation quirks, edge cases, questions / issues)
-

#### Manual testing tips
-

#### Tag anyone who definitely needs to review or help
-

### CODE REVIEW

File: pt_miniscreen/tiles/menu/network.py [hunk 1]
```diff
@@ -8,6 +8,7 @@ def __init__(self, size, pos=(0, 0)):
             size=size,
             pos=pos,
             pages=[
+                LoginPage,
```
Comment (olivierwilkinson) [inline pt_miniscreen/tiles/menu/network.py:11]: I think the login page is meant to be the first page in the System menu.

**Review by olivierwilkinson — CHANGES_REQUESTED** [review]
Looks great, particularly like that you made sure to hide the password when when it's not the default. The page is not meant to be in the Network menu though, it's meant to be in the System menu now.

**Review by olivierwilkinson — APPROVED** [review]
LGTM!
